src/components/Guest/Guest.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+type Props = {
+  id: number;
+  guestName: string;
+  phoneNumber: number;
+  deleteGuest(id: number): void
+}
+const Guest: React.FC<Props> = ({
+  guestName,
+  id,
+  phoneNumber,
+  deleteGuest
+}) => {
+  return (
+    <article className="guestCard">
```

Comment:
Looks great! Let's get this re-routed so that it shows up inside of the wedding details view instead of in it's own path

Reply:
^ Kristi has a good idea with the re-route. Awesome job getting the delete functionality working with TS